__tests__/feature/dashboard.test.js

```diff
@@ -28,7 +28,7 @@ describe("viewing the dashboard", () => {
       await screen.findByText(/Uber template1/)
 
       // Click the resource template
-      fireEvent.click(screen.getByText("Uber template1", { selector: "a" }))
+      fireEvent.click(screen.getByTestId("createResource-Uber template1"))
```

Comment:
Dobbs' says 
> it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible

https://testing-library.com/docs/queries/bytestid

Have you exhausted all other options both here and in all following tests?

Reply:
I couldn't find another way of getting to this link, since it is no longer a text link but an icon with a generic `title` attribute (and there could be multiple of these links on the page).  I thought of giving the link a unique `id`, but noticed the `data-testid` pattern used extensively in other Sinopia tests, and also found this article by Kent C Dodds discussing it: https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change   But perhaps I missed a selector possibility.

Reply:
It seems to me that the lack of a unique title for this icon is an accessibility problem.  How is a screen reader user to know which type they are creating when they activate the button? Is there a reason we can't give it a unique title?  Something like  `title="Create resource ${row.resourceLabel}"` ?

Reply:
Makes sense -- my implementation was just following the exact specs of the ticket, but this seems totally reasonable.  And also for the Copy and Edit buttons (which will then require more test selector updates)